janitor/functions.py

```diff
@@ -2054,19 +2054,21 @@ def collapse_levels(df: pd.DataFrame, sep: str = "_"):
 
     :param df: A pandas DataFrame.
     :param sep: String separator used to join the column level names
+    :param axis: 0 for index and 1 for columns 
     :returns: df
     """
 
     check("sep", sep, [str])
-
     # if already single-level, just return the DataFrame
+    df= df.transpose() if axis==0 else df
```

Comment:
@mdini code style suggestion here:

```suggestion
    df = df.transpose() if axis==0 else df
```